src/activate.ts

```diff
@@ -27,9 +28,23 @@ async function Activate(): Promise<void> {
                 const servicesConfig = core.getInput('services-config', { required: true });
                 await licenseClient.ActivateLicenseWithConfig(servicesConfig);
             } else {
-                const username = core.getInput('username', { required: true }).trim();
-                const password = core.getInput('password', { required: true }).trim();
-                const serial = core.getInput('serial', { required: license.toLowerCase().startsWith('pro') });
+                const pro = license.toLowerCase().startsWith('pro');
+                let username = core.getInput('username', { required: pro }).trim();
+                let password = core.getInput('password', { required: pro }).trim();
```

Comment:
this would still throw an error even if we tried to read env var

Reply:
wasn't intending to support env var for pro